roslibrust_codegen/src/lib.rs

```diff
@@ -127,7 +155,11 @@ fn parse_message_files(
             }
         }
     }
-    Ok(resolve_message_dependencies(parsed_messages))
+    parsed_services.sort_by(|a, b| a.name.cmp(&b.name));
```

Comment:
Making sure I understand the reasoning: sorting here so that they come out alphabetized in the final source output right?

Reply:
Yup I wrote an CI test to catch that generation had been re-run after I forgot to do it, but then the CI test failed because the output came out differently ordered on my system vs CI.